### ISSUE TITLE
OData Connected Service in VB.NET Solution

### ISSUE THREAD

**Issue by wurst0815** [OData/odata.net#1206]
Hi,

when will the OData Connected Service Feature be available in VB.NET Solutions?
See issue I've opened here.
https://github.com/OData/lab/issues/78

Thanks in advance.

**Comment by wurst0815** [comment]
Yes issue https://github.com/OData/lab/issues/78 Is specific to Vb.net

C# works fine

**Comment by biaol-odata** [comment]
@wurst0815 Sorry I really don't understand what you mean exactly. Do you mean both issues (https://github.com/OData/odata.net/issues/1206 and https://github.com/OData/lab/issues/78) are specific for VB.NET, and this issue is a duplicate?

**Comment by wurst0815** [comment]
https://github.com/OData/lab/issues/78 Was the first issue. At that point in time I did not know that it Is specific to vb.net

But as nobody seems to Take care of the issue I thought it Is not monitored anymore and so I created https://github.com/OData/odata.net/issues/1206

**Comment by biaol-odata** [comment]
@wurst0815 OK. I have transported relevant information to the https://github.com/OData/lab/issues/78, where this issue should be tracked.

**Comment by wurst0815** [comment]
Ok. Would really be great if someone could take care of it.
As you can see I am not the only one waiting for a solution.

**Comment by biaol-odata** [comment]
@wurst0815 looks like you've create another issue #1207 to track. Can this one be closed as duplicate?

**Comment by wurst0815** [comment]
No, one is related to ConnectedService, the other one to Client Code Generator. For me two different issues

**Comment by unchase** [comment]
Since v0.5.0 my [Unchase OData ConnectedService](https://marketplace.visualstudio.com/items?itemName=Unchase.unchaseodataconnectedservice) is supports `VisualBasic` client-code generation.
